subsys/mpsl/clock_ctrl/mpsl_clock_ctrl.c

```diff
@@ -304,6 +304,14 @@ static int32_t m_lfclk_release(void)
  */
 static struct nrf_clock_spec m_lfclk_specs;
 
+/* Minimum allowed HFXO startup time in microseconds. Use conservative value that comes from BICR.
+ *
+ * The time is used by MPLS to request the clock before radio event start. If the time is too short
+ * there will be no time run low priority thread to handle Zephyr PM subsystem configuration of
+ * no-latency event and request the MRAM to be always on.
+ */
+#define MPLS_PM_HFCLK_MINIMUM_ALLOWED_STARTUP_TIME_US 850
```

Comment:
Please fix the typo `MPLS -> MPSL`

Reply:
Ah, I missed one in comment :/. Sorry, I should have checked the whole file in the first place, instead just searching for macro.